alf/algorithms/algorithm.py

```diff
@@ -1289,13 +1286,16 @@ def train_from_replay_buffer(self, update_global_counter=False):
         """
         config: TrainerConfig = self._config
 
-        if self._replay_buffer.total_size < config.initial_collect_steps:
-            # returns 0 if haven't started training yet; throughput will be 0
+        # returns 0 if haven't started training yet, when ``_replay_buffer`` is
+        # not None and the number of samples in the buffer is less than
+        # ``initial_collect_steps``; throughput will be 0 in this phase.
+        if (self._replay_buffer and
```

Comment:
because this is inside ``train_from_replay_buffer``, then the replay buffer should always not be None?

Reply:
> because this is inside `train_from_replay_buffer`, then the replay buffer should always not be None?

In the case that online RL training is not started yet, ``_replay_buffer`` is None.